examples/cp4ba/variables.tf

```diff
@@ -1,7 +1,10 @@
+variable "ibmcloud_api_key" {
+  description = "Enter your IBM API Cloud access key. Visit this link for more information: https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui "
+}
 
-variable "cluster_name_or_id" {
+variable "cluster_id" {
   default     = ""
-  description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
+  description = "Set your cluster ID to install the Cloud Pak for Business Automation. Leave blank to provision a new OpenShift cluster."
```

Comment:
This is required.  The example won't provision a cluster.  It needs an existing cluster to install.

Reply:
I fixed it. 
```
variable "cluster_id" {
  default     = ""
  description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
}
```